### DISCUSSION TITLE
Why does kitty fail to show my prompt?

### DISCUSSION THREAD

**Discussion by rew1nter** [kovidgoyal/kitty#7288] [Q&A] [answered]
As you can see both Alacritty and Konsole shows the prompt correctly, but kitty can't for some reason.
![image](https://github.com/kovidgoyal/kitty/assets/64508791/7de30d02-a5d3-4317-ab57-0c8838d9dfb6)

**Comment by kovidgoyal** [comment, marked as answer]
Because you have something in your shell rc files that's interfering
with kitty shell integration. Bisect it to find out what. Or turn off
shell integration in kitty.
